File: pinyougou-parent/pinyougou-interface/src/main/java/cn/itcast/core/service/OrderService.java
package cn.itcast.core.service;

import cn.itcast.core.pojo.order.Order;

import entity.PageResult;
import entity.SearchOrderParam;
import vo.OrderVo;

public interface OrderService {
    void add(Order order);

    PageResult search(Integer page, Integer rows, SearchOrderParam searchOrderParam, String name);

    void updateStatus(Long[] ids);

    PageResult search(Integer page, Integer rows, OrderVo orderVo);


    Order findOne(Long orderId);


    void delete(Long[] ids);


}

File: pinyougou-parent/pinyougou-web-shop/src/main/webapp/js/service/orderService.js
//服务层
app.service('orderService',function($http){
	    	
	//读取列表数据绑定到表单中
	this.findAll=function(){
		return $http.get('../order/findAll.do');
	}

    //搜索
    this.search=function(page,rows,searchEntity){
        return $http.post('../order/search.do?page='+page+"&rows="+rows, searchEntity);
    }
    //修改发货状态
    this.updateStatus = function(ids){
        return $http.get('../order/updateStatus.do?ids='+ids);
    }
});
